### PR TITLE
Bar & Histogram & Formula  & ComparativeFormula Widgets: Add a skeleton for loading state

### PR DESCRIPTION
**Pull request by vmilan**
# Description

Shortcut: https://app.shortcut.com/cartoteam/story/314283/new-loader-ui-for-widgets
[sc-314283]

### CODE REVIEW

File: packages/react-ui/src/theme/sections/components/feedback.js [hunk 1]
```diff
@@ -7,5 +10,30 @@ export const feedbackOverrides = {
         horizontal: 'center'
       }
     }
+  },
+
+  // Skeleton
+  MuiSkeleton: {
+    defaultProps: {
```
Comment (VictorVelarde) [inline packages/react-ui/src/theme/sections/components/feedback.js:17]: Ok, good to know we have some basics in common

File: packages/react-widgets/src/widgets/BarWidget.js [hunk 2]
```diff
@@ -156,6 +156,7 @@ function BarWidget({
             height={height}
             animation={animation}
             filterable={filterable}
+            isLoading={isLoading}
```
Comment (VictorVelarde) [inline packages/react-widgets/src/widgets/BarWidget.js:159]: Why is this injected externally? Shouldn't this be just a prop that the 'widget' launching the query, passes down to the UI?

Comment (vmilan) [reply packages/react-widgets/src/widgets/BarWidget.js:159]: This is what I'm doing here, isn't it? passing down the prop from the parent to the UI, as in line 137 for the WrapperWidgetUI.

The logic was already there, but the UI wasn't receiving it.

**Comment by vmilan** [comment]
@VictorVelarde added also the HistogramWidget

File: packages/react-widgets/src/widgets/BarWidget.js (hunk 2 above)
Comment (VictorVelarde) [reply packages/react-widgets/src/widgets/BarWidget.js:167]: Yes, misunderstanding on my side. It's fine

**Pull request merged**